library/Vanilla/Web/Controller.php

```diff
@@ -27,7 +27,7 @@
     /**
      * @var SessionInterface
      */
-    private $session;
+    protected $session;
```

Comment:
This should remain private I think,
`getSession()` should be used to access it. We probably do not want child classes to modify the variable.